<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtn_principal</class>
 <widget class="QMainWindow" name="vtn_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ingreso Persona</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_nombre">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_nombre">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_grabar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Grabar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_apellido">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_apellido">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_tipo_persona">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Docente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Estudiante</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_tipo_persona">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cedula">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cedula:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_cedula">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_email">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_email">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar_cedula">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar por cedula</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mnb_menu_principal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="stb_estado"/>
 </widget>
 <resources/>
 <connections/>
</ui>
